networking/libsnnet/cn_test.go

```diff
@@ -748,3 +786,62 @@ func TestCN_Whitebox(t *testing.T) {
 	assert.Nil(vnic.Enable())
 	assert.Nil(bridge.Enable())
 }
+
+//Whitebox test the CN API
+//
+//This tests exercises tests the primitive operations
+//that the CN API rely on. This is used to check any
+//issues with the underlying netlink library or kernel
+//This tests fails typically if the kernel or netlink
+//implementation changes
```

Comment:
Can you add something indicating this is an OVS test specifically for this comment?